src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
```

Comment:
```suggestion
    private int radius;
```
radius doesn't need to be final